theorems/T000294.md

```diff
@@ -0,0 +1,9 @@
+---
+uid: T000294
+if:
+  P000052: true
+then:
+  P000136: true
+---
+
+Infinite subsets are {P52} and therefore not {P16}.

```

Comment:
I am confused by this sentence.  Maybe you meant something like "Infinite subsets of a discrete space are not compact".